fabric-dynamic-registry-api-v1/src/main/java/net/fabricmc/fabric/api/dynamicregistry/v1/DynamicRegistryProvider.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.dynamicregistry.v1;
+
+import java.util.List;
+
+public interface DynamicRegistryProvider {
+	void getDynamicRegistries(List<DynamicRegistryEntry<?>> entries);
```

Comment:
Hmm, if we only add to the list, I'd make it
```java
	void getDynamicRegistries(Consumer<DynamicRegistryEntry<?>> adder);
```

Reply:
Fixed in b772f1429ff4953e51da1de9b520dfcba3434e1b